src/test/java/uk/gov/register/configuration/RegistersConfigurationFunctionalTest.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.register.configuration;
+
+import org.junit.Test;
+import uk.gov.register.core.RegisterMetadata;
+import uk.gov.register.core.RegisterName;
+
+import java.util.Optional;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.fail;
+
+public class RegistersConfigurationFunctionalTest {
```

Comment:
I don't think these are really functional tests so can you remove Functional from the name?